marketpulse/auth/admin.py

```diff
@@ -1,14 +1,15 @@
 from django.contrib import admin
-from django.contrib.auth.models import User
 
 from import_export import resources
 from import_export.admin import ExportMixin
 
+from marketpulse.auth.models import User
```

Comment:
You could use [django.contrib.auth.get_user_model()](https://docs.djangoproject.com/en/1.7/topics/auth/customizing/#referencing-the-user-model) if you don't want to hardcode the custom User model.